### DISCUSSION TITLE
为什么显卡驱动的教程要滞后于gui的安装？

### DISCUSSION THREAD

**Discussion by aswirl4** [NakanoMikuOrg/arch-guide#42] [Q&A]
新手用2台电脑照着这样的顺序先安装了gui。
其中一台电脑在gui使用过程中会随机崩溃，返回sddm登陆页面， 丢失所有工作进度。
另一台则是sddm登陆之后就花屏/黑屏。
所以请问顺序为什么不先安装显卡驱动再gui呢？

**Comment by ptazithos** [comment]
方便把硬件配置发出来看一下吗？

**Reply by aswirl4** [reply]
> 方便把硬件配置发出来看一下吗？

使用过程随机崩溃然后返回sddm的是：
i7-9750h
GTX 1660ti
8G*2
nvme 1T

另一个sddm登录后直接花屏黑屏的是:
E5 2689
R5 220
16G*2 ECC
nvme 500G

**Reply by ptazithos** [reply]
抱歉，看不出什么由头。
可能需要补充一下崩溃的系统日志。

一般来讲，没有图形驱动的情况下，开启桌面也就仅仅是没有硬件加速而已。
除了卡一些，应该不会有直接崩溃的情况。

那你的花屏闪退的问题在安装驱动后，有修复吗。
如果没有问题的话，我去补充一个提醒到安装桌面的步骤里。

**Reply by aswirl4** [reply]
> 抱歉，看不出什么由头。 可能需要补充一下崩溃的系统日志。
> 
> 一般来讲，没有图形驱动的情况下，开启桌面也就仅仅是没有硬件加速而已。 除了卡一些，应该不会有直接崩溃的情况。
> 
> 那你的花屏闪退的问题在安装驱动后，有修复吗。 如果没有问题的话，我去补充一个提醒到安装桌面的步骤里。

数月之前的事情了，日志已丢失。
2者均在我手动安装显卡驱动之后修复了。

**Reply by ptazithos** [reply]
那只能先这样了
增加了关于桌面环境异常的提示 #58
